src/web/app/svgedit/interaction/mouseInteractions.ts

```diff
@@ -364,7 +373,8 @@ const mouseDown = (evt: MouseEvent) => {
 
           if (!rightClick) {
             if (evt.altKey) {
-              const cmd = clipboard.cloneSelectedElements(0, 0, { addToHistory: false })?.cmd;
+              const cmd = (await clipboard.cloneSelectedElements(0, 0, { addToHistory: false }))
+                ?.cmd;
```

Comment:
Optional:
would be better to keep mousedown synchronous if possible. But this would not take too much time so should be fine right now.